api/capi/meson.build

```diff
@@ -94,6 +81,57 @@ nnstreamer_capi_dep = declare_dependency(link_with: nnstreamer_capi_lib,
   ]
 )
 
+# Create dependency for tensor_filter_single for single-shot capi
+tensor_filter_single_lib = shared_library ('tensor_filter_single_lib',
+  join_paths(meson.current_source_dir(), 'src', 'tensor_filter_single.c'),
+  dependencies: [nnstreamer_dep, glib_dep, gobject_dep, gst_dep],
+  include_directories: [inc, nninc],
+  install: true,
+  version: meson.project_version(),
+  install_dir: nnstreamer_libdir,
+)
+
+tensor_filter_single_dep = declare_dependency(link_with: tensor_filter_single_lib,
+  include_directories: [inc],
+)
+
+# New single-shot c-api
+capi_single_new_main = []
+capi_single_new_main += join_paths(meson.current_source_dir(), 'src', 'nnstreamer-capi-pipeline.c')
+capi_single_new_main += join_paths(meson.current_source_dir(), 'src', 'nnstreamer-capi-util.c')
+capi_single_new_main += join_paths(meson.current_source_dir(), 'src', 'nnstreamer-capi-single-new.c')
+
+nnstreamer_capi_single_new_lib = shared_library ('capi-nnstreamer-single-new',
+  capi_single_new_main,
+  dependencies: [capi_deps, tensor_filter_single_dep],
+  include_directories: [
+    inc,
+    nninc
+  ],
```

Comment:
how about declaring a variable for `include_directories` (because it's repeated) or using the same format with line 88?